.storybook/config.js

```diff
@@ -1,3 +0,0 @@
-import { configure } from '@storybook/react';
-
-configure(require.context('../src', true, /\.stories\.tsx$/), module);
```

Comment:
Is the configure from `@storybook/react` not needed then?

Reply:
No not anymore 👍